webapp/crashstats/crashstats/management/commands/verifyprocessed.py

```diff
@@ -163,6 +164,8 @@ def handle_missing(self, date, missing):
                         pass
                     else:
                         raise
+                reprocessing_api.post(crash_ids=crash_id)
```

Comment:
Given that these were never processed, it's probably the case we should put them in the standard queue. However, it doesn't really matter and we have an API for reprocessing, so I opted to reuse that.